spiffworkflow-backend/src/spiffworkflow_backend/api.yml

```diff
@@ -2091,6 +2091,26 @@ paths:
               schema:
                 $ref: "#/components/schemas/Task"
 
+  /tasks/prepare_form:
```

Comment:
i think we normally dasherize the paths.

Reply:
fixed